JASP-Engine/JASP/R/ttestbayesianpairedsamples.R

```diff
@@ -543,23 +543,29 @@ TTestBayesianPairedSamples <- function(dataset=NULL, options, perform="run", cal
 					
 					})
 					
-					
-					if (class(result) == "try-error") {
-						
-						errorMessage <- .extractErrorMessage(result)
-						
-						if (errorMessage == "x or y must not contain missing or infinite values.") {
-							
-							errorMessage <- paste("Bayes factor is undefined - one or both of the variables contain infinity")
-							
-						} else if (errorMessage == "data are essentially constant") {
-							
-							errorMessage <- paste("Bayes factor is undefined - the sample contains all the same value (zero variance)")
-							
-						} else if (errorMessage == "Insufficient sample size for t analysis." || errorMessage == "not enough observations") {
-							
-							errorMessage <- "Bayes factor is undefined - one or both of the variables has too few observations (possibly only after missing values are excluded)"	
-						}
+					errors <- .hasErrors(dataset, perform, message = 'short', type = c('infinity','observations','variance'),
```

Comment:
Above the analysis. If we know we cannot calculate the model, we shouldn't try to.
Also probably the trycatch thing again.